src/test/DummyRemoteSimulator.ts

```diff
@@ -152,7 +152,6 @@ export class DummyRemoteSimulator extends RemoteSimulator {
             this.frameCounter = frameNumber;
 
             const msg: NetMessage = JSON.parse(this.getDataBundle(frameNumber));
-            this.frameCounter;
```

Comment:
what happened here?

Reply:
one of our ci tests was flagging that as an expression that does nothing, so I removed it.  I have no idea how it got there in the first place.